mmv1/third_party/terraform/services/storage/data_source_google_storage_buckets_test.go

```diff
@@ -0,0 +1,111 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+package storage_test
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-provider-google/google/acctest"
+	"github.com/hashicorp/terraform-provider-google/google/envvar"
+)
+
+func TestAccDataSourceGoogleStorageBuckets_basic(t *testing.T) {
+	t.Parallel()
+
+	project := envvar.GetTestProjectFromEnv()
```

Comment:
looking more into this is one of those cases where we'd want to also create a project as part of the test so that the environment stays consistent. This makes it so that only the two buckets that are created here will only exist for this project that's also created as part of running the test.

This is done in other resources, an example of this is found here: 
https://github.com/GoogleCloudPlatform/magic-modules/blob/5c9241ea6e575e05912706b024f9acdbf47be82d/mmv1/third_party/terraform/services/storage/resource_storage_bucket_test.go.erb#L2226-L2231

Reply:
This is a really good approach.
Thank you very much!

I'd like to implement it, but I'm restricted to a single project and cannot run the tests locally then.